### PR TITLE
Feature/project management/287 aql enhancement

### PR DESCRIPTION
**Pull request by stefanspiska**

### CODE REVIEW

File: client/src/test/java/org/ehrbase/client/aql/condition/ConditionTest.java [hunk 1]
```diff
@@ -42,13 +43,83 @@ public void testBuildAql() {
         containmentObservation.bindQuery(query);
         Condition condition1 = Condition.greaterOrEqual(containmentObservation.DIASTOLIC_MAGNITUDE, 13d);
         Condition condition2 = Condition.notEqual(containmentObservation.MEAN_ARTERIAL_PRESSURE_UNITS, "mh");
-        Condition condition3 = Condition.lessThen(containmentObservation.TIME_VALUE, OffsetDateTime.of(2019, 04, 03, 22, 00, 00, 00, ZoneOffset.UTC));
+        Condition condition3 = Condition.lessThan(containmentObservation.TIME_VALUE, OffsetDateTime.of(2019, 04, 03, 22, 00, 00, 00, ZoneOffset.UTC));
 
-        Condition cut = Condition.and(condition1, Condition.or(condition2, condition3));
+        //Condition cut = Condition.and(condition1, Condition.or(condition2, condition3));
```
Comment (jakesmolka) [inline client/src/test/java/org/ehrbase/client/aql/condition/ConditionTest.java:48]: Remove or ToDo?

File: client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/CoronaTestIT.java [hunk 2]
```diff
@@ -68,21 +72,139 @@ public void testCorona() throws IOException {
 
         assertThat(openEhrClient.compositionEndpoint(ehr).find(coronaAnamneseComposition.getVersionUid().getUuid(), CoronaAnamneseComposition.class)).isNotNull();
 
-
+/*
```
Comment (jakesmolka) [inline client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/CoronaTestIT.java:75]: Remove or ToDo?

File: client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/DefaultRestAqlEndpointTestIT.java [hunk 3]
```diff
@@ -158,6 +162,205 @@ public void testExecuteEntityQuery() {
...
+        }
+
+        List<TestCase> testCases = new ArrayList<>();
+
+
+        testCases.add(new TestCase(1,
+                TopExpresion.forward(1),
+                comp3.getVersionUid().getUuid()));
+
+        /* Direction  is ignored on ehrbase
```
Comment (jakesmolka) [inline client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/DefaultRestAqlEndpointTestIT.java:337]: So this is a ToDo?

Comment (stefanspiska) [reply client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/DefaultRestAqlEndpointTestIT.java:337]: yes see https://github.com/ehrbase/ehrbase/issues/265

File: client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/CoronaTestIT.java (hunk 2 above)
Comment (stefanspiska) [reply client/src/test/java/org/ehrbase/client/openehrclient/defaultrestclient/CoronaTestIT.java:75]: remove

**Comment by jakesmolka** [comment]
Note: CI pipeline result (fail) is WIP and can be ignored at the current state.

**Pull request merged**